node/core/approval-voting/src/lib.rs

```diff
@@ -697,7 +697,9 @@ fn approval_signing_payload(
 	approval_vote: ApprovalVote,
 	session_index: SessionIndex,
 ) -> Vec<u8> {
-	(approval_vote, session_index).encode()
+	const MAGIC: [u8; 4] = *b"APPR";
```

Comment:
Not sure if this is worth a type yet.

Reply:
In what way is it different?

Reply:
Strike that, it would only different for enums.